### PR TITLE
[ru] Localization of /concepts/scheduling-eviction/api-eviction.md

### PR DESCRIPTION
**Pull request by kirkonru**
Add RU localization for api-eviction.md and _index.md.

### CODE REVIEW

**Comment by kirkonru** [comment]
@shurup hello! Please review and merge this PR.

**Review by shurup — CHANGES_REQUESTED** [review]
Thank you for this PR! Here are a few minor corrections for me to move on with it.

File: content/ru/docs/concepts/scheduling-eviction/_index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  В Kubernetes под планированием понимается поиск подходящих узлов, на которых kubelet сможет запустить Pod'ы. 
+  Приоритизация — процесс завершения работы Pod'ов с более низким приоритетом и высвобождения места для Pod'ов 
+  с более высоким приоритетом. Вытеснение — это проактивное завершение работы одного или нескольких Pod'ов на 
+  узлах с дефицитом ресурсов.
+
+no_list: true
+---
+
+В Kubernetes под планированием понимается поиск {{<glossary_tooltip text="узлов" term_id="node">}}, подходящих для размещения {{<glossary_tooltip text="Pod'ов" term_id="pod">}} так, чтобы {{<glossary_tooltip text="kubelet" term_id="kubelet">}} 
+мог их запустить. Приоритизация (упреждение) — процесс завершения работы Pod'ов
```
[the remark this answers is not in the document]
Comment (kirkonru) [reply content/ru/docs/concepts/scheduling-eviction/_index.md:15]: Great suggestion, thanks!

File: content/ru/docs/concepts/scheduling-eviction/api-eviction.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+для этого Pod'а, которая контролируется политикой. 
+
+## Вызов API Eviction
+
+Для доступа к API Kubernetes и создания объекта `Eviction` можно воспользоваться [клиентской библиотекой](/docs/tasks/administer-cluster/access-cluster-api/#programmatic-access-to-the-api). Необходимая операция оформляется в виде POST-запроса (см. пример ниже):
+
+{{< tabs name="Eviction_example" >}}
+{{% tab name="policy/v1" %}}
+{{< note >}}
+Вытеснение с версией `policy/v1` доступно, начиная с v1.22. Для более ранних релизов используйте `policy/v1beta1`.
```
[the remark this answers is not in the document]
Comment (kirkonru) [reply content/ru/docs/concepts/scheduling-eviction/api-eviction.md:28]: Thanks a lot! The Russian language is so confusing sometimes. :)

File: content/ru/docs/concepts/scheduling-eviction/api-eviction.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
+---
+title: Вытеснение, инициированное через API
+content_type: concept
+weight: 110
+---
+
+{{< glossary_definition term_id="api-eviction" length="short" >}} </br>
```
Comment (kirkonru) [inline content/ru/docs/concepts/scheduling-eviction/api-eviction.md:7]:
```suggestion
{{< glossary_definition term_id="api-eviction" length="short" >}} </br>
```

File: content/ru/docs/reference/glossary/pod-disruption.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+ - operation
+---
+
+[Нарушение работы Pod'ов (Pod disruption)](/docs/concepts/workloads/pods/disruptions/) — процесс, 
+в ходе которого происходит плановое или внеплановое (принудительное) завершение работы Pod'ов на узлах. 
+
+<!--more--> 
+
+Плановое завершение работы Pod'ов инициируется владельцами приложений или администраторами
+кластера. Внеплановое завершение работы обычно связано с непредвиденными проблемами различной природы,
```
Comment (kirkonru) [inline content/ru/docs/reference/glossary/pod-disruption.md:23]:
```suggestion
кластера. Внеплановое завершение работы обычно вызвано непредвиденными обстоятельствами различной природы, 
```

File: content/ru/docs/reference/glossary/api-eviction.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+- operation
+---
+  Вытеснение, инициированное через API — процесс, при котором с помощью [Eviction API](/docs/reference/generated/kubernetes-api/{{<param "version">}}/#create-eviction-pod-v1-core) 
+создается объект `Eviction`, который запускает корректное завершение работы Pod'а.
+
+<!--more-->
+
+Вытеснение можно запросить через Eviction API, обратившись к нему напрямую, либо программно (через клиент API-сервера — например, с помощью команды `kubectl drain`). При этом будет создан объект `Eviction`, на основании которого API-сервер завершит работу Pod'а.
+
+Инициированные API вытеснения учитывают заданные параметры [`PodDisruptionBudget`](/docs/tasks/run-application/configure-pdb/) (минимальное количество реплик, которые должны быть доступны для данного развертывания в любой момент времени) и [`terminationGracePeriodSeconds`](/docs/concepts/workloads/pods/pod-lifecycle#pod-termination) (период ожидания корректного завершения работы Pod'а).
```
Comment (kirkonru) [inline content/ru/docs/reference/glossary/api-eviction.md:20]:
```suggestion
Вытеснения, инициированные через API, учитывают заданные параметры [`PodDisruptionBudget`](/docs/tasks/run-application/configure-pdb/) (минимальное количество реплик, которые должны быть доступны для данного развертывания в любой момент времени) и [`terminationGracePeriodSeconds`](/docs/concepts/workloads/pods/pod-lifecycle#pod-termination) (период ожидания корректного завершения работы Pod'а).
```

**Comment by kirkonru** [comment]
@shurup appreciate the review, all the corrections are committed. I had to add two more files to the Glossary to get rid of the following errors:

> ERROR 2023/02/16 12:13:24 [ru] "docs/concepts/scheduling-eviction/api-eviction.md": "api-eviction" is not a valid glossary term_id, see ./docs/reference/glossary/* for a full list
> 
> ERROR 2023/02/16 13:03:30 [ru] "docs/concepts/scheduling-eviction/_index.md": "pod-disruption" is not a valid glossary term_id, see ./docs/reference/glossary/* for a full list

Now the tests run smoothly, and the PR seems to be ready for merging (well, at least I hope so :smile:)

**Comment by shurup** [comment]
Great, thanks! All looks fine to me.

/lgtm
/approve

**Pull request merged**